src/hooks/abrt_exception_handler3.py.in

```diff
@@ -68,6 +70,8 @@ def send(data):
 
 
 def write_dump(tb_text, tb):
+    import os
```

Comment:
There is no point to import os lazily. The module is always imported, even with ``-S`` (don't imply 'import site' on initialization).

Reply:
```
$ python3 -S -c "import sys; print('os' in sys.modules)"
False
```

Reply:
But my main motivation for lazy importing `os` was that *maybe* some day in the future, we'll avoid it in default Python startup and this makes `abrt_exception_handler3` prepared for that future.

Reply:
I would even lazy import `sys`, however `sys.excepthook` is actually set on import time.